### PR TITLE
Milestone: Created a SPA with React and Redux

### PR DESCRIPTION
**Pull request by AngomRanjan**
#### In this milestone:

- Created a Component The Retrieves and Data from an External API.
- Created Another Component to Fetch Detail Information.
- Created action and reducer to fetch data from API and store it in the store.
- Written Unit Test for pure functions.
- [Link to video presentation](https://www.loom.com/share/59358e0d86054813a5687fdc44ac7ffd)
- [Link to Live Demo](https://stay-tune-angom.netlify.app/)

- Special Thanks To Nelson on Behance for the original design idea

### CODE REVIEW

File: src/css/Show.css [hunk 1]
```diff
@@ -0,0 +1,41 @@
+/* Show Stylesheet */
+
```
Comment (rloterh) [inline src/css/Show.css:2]: - [ ] OPTIONAL
 In as much as we want your design to closely mimic the given [design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), your background color matches your design, and so is the font. However, I think you can still implement the dual alternating color pattern for the cells on the mobile screen, as seen in the provided template.

File: README.md [hunk 2]
```diff
@@ -2,19 +2,40 @@
...
+navigate to the project directory, from GitHub,
+to test or run the application double click on the `index.html` file to open in your browser
 
 ## Live Demo
 
 [Live Demo Link](https://stay-tune-angom.netlify.app)
 
+[Project Presentation Video](https://www.loom.com/share/59358e0d86054813a5687fdc44ac7ffd)
+
 ## Authors
```
Comment (rloterh) [inline README.md:45]: - [ ] OPTIONAL
I think you should change the header title _Authors_ to _Author_ in your ReadMe, as this project is worked on by sole contributor.

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Angom,  

Your project is complete, awesome job🥇! There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉🌟 

## Highlights 👍
- Project is properly deployed
- SPA feature correctly implemented
- State management well implemented
- Good unit test coverage with react testing library

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

_Kindly consider my comments marked **[OPTIONAL]** under this review_
<br>
Keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (AngomRanjan) [reply README.md:45]: Thank you, I will change it right away

**Pull request merged**